zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -28,14 +28,7 @@
 import java.net.SocketTimeoutException;
 import java.net.URISyntaxException;
 import java.net.UnknownHostException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Date;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Hi @zhugezifang ,  It's not recommended to use a wild card in the `import` statement

Reply:
> Hi @zhugezifang , It's not recommended to use a wild card in the `import` statement

ok,thanks your advice,i will change it